src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -317,6 +317,13 @@ export default class GroupComparisonPage extends React.Component<
         },
     });
 
+    @autobind private isGroupDeletable() {
```

Comment:
should this be computed?

Reply:
it's implemented as a callback to fit the interface of GroupSelector